tutorials/desktop/decoding-log-files/decoding-log-files.md

```diff
@@ -0,0 +1,177 @@
+Duration: 2:00
+
+There are many different log files that all serve different functions. When trying to find a log about something, you should start by choosing the most relevant log file. Below is a list of some common log file locations.
+
+### System logs
+
+System logs deal with exactly that - the Ubuntu system - as opposed to extra applications added by the user. These logs may contain information about authorization, system daemons and system messages.
+
+#### Authorization log
+
+Location: `/var/log/auth.log`
+
+Keeps track of authorization systems, such as password prompts, the `sudo` command and remote logins.
+
+#### Daemon Log
+
+Location: `/var/log/daemon.log`
+
+Daemons are programs that run in the background, usually without user interaction. For example, display server, ssh sessions, printing services, bluetooth, and more.
+
+#### Debug log
+
+Location: `/var/log/debug`
+
+Provides debugging information from the Ubuntu system and applications.
+
+#### Kernel log
+
+Location: `/var/log/kern.log`
+
+Logs from the Linux kernel.
+
+#### System log
+
+Location: `/var/log/syslog`
+
+Contains more information about your system. If you can't find anything in the other logs, it's probably here.
+
+### Application logs
+
+Some applications also create logs in `/var/log`. Below are some examples.
+
+#### Apache logs
+
+Location: `/var/log/apache2/` (subdirectory)
+
+Apache creates several log files in the `/var/log/apache2/` subdirectory. The `access.log` file records all requests made to the server to access a file. `error.log` records all errors thrown by the server.
+
+#### X11 server logs
+
+Location: `/var/log/Xorg.0.log`
+
+The X11 server creates a seperate log file for each of your displays. Display numbers start at zero, so your first display (display 0) will log to `Xorg.0.log`. The next display (display 1) would log to `Xorg.1.log`, and so on.
+
+
+### Non-human-readable logs
+
+Not all log files are designed to be read by humans. Some were made to be parsed by applications. Below are some of examples.
+
+#### Login failures log
+
+Location: `/var/log/faillog`
+
+Contains info about login failures. You can view it with the `faillog` command.
+
+#### Last logins log
+
+Location: `/var/log/lastlog`
+
+Contains info about last logins. You can view it with the `lastlog` command.
+
+#### Login records log
+
+Location: `/var/log/wtmp`
+
+Contains login info used by other utilities to find out who's logged in. To view currently logged in users, use the `who` command.
+
+positive
+: **This is not a complete list!**
+You can search the web for more locations relevant to what you're trying to debug. There is also a slightly more complete list [here](https://help.ubuntu.com/community/LinuxLogFiles#System_Logs).
+
+## Viewing logs using GNOME System Log Viewer
+Duration: 3:00
+
+The GNOME System Log Viewer provides a simple GUI for viewing and monitoring log files. It comes pre-installed with Ubuntu under the name `System Log`.
```

Comment:
Maybe add that on Ubuntu 17.10 and above, it's called "Logs"

**Bold**, not `code` for app names.